src/pybind/mgr/dashboard/README.rst

```diff
@@ -13,8 +13,8 @@ Enabling and Starting the Dashboard
 
 If you want to start the dashboard from within a development environment, you
 need to have built Ceph (see the toplevel ``README.md`` file and the `developer
-documentation <http://docs.ceph.com/docs/master/dev/>`_ for details on how to
-accomplish this.
+documentation <https://docs.ceph.com/docs/master/dev/developer_guide/>`_ for
```

Comment:
This could be more specific using [1]? 
[1] https://docs.ceph.com/docs/master/dev/developer_guide/#building-from-source

Reply:
Yes, fine with me. Thanks for the suggestion @joscollin .